commands/show-beacons.go

```diff
@@ -3,195 +3,97 @@ package commands
 import (
 	"fmt"
 	"os"
-	"sort"
-	"strings"
 	"text/template"
 
-	"gopkg.in/mgo.v2/bson"
-
 	"github.com/ocmdev/rita/config"
+	"github.com/ocmdev/rita/database"
+	"github.com/ocmdev/rita/datatypes/TBD"
 	"github.com/urfave/cli"
 )
 
-var beaconHeader = false
+var cutoffScore float64
 
 func init() {
 	command := cli.Command{
 		Name:  "show-beacons",
 		Usage: "print beacon information to standard out",
 		Flags: []cli.Flag{
-			cli.BoolFlag{
-				Name:        "no-header, r",
-				Usage:       "turn off the header row",
-				Destination: &beaconHeader,
-			},
 			humanFlag,
 			databaseFlag,
+			cli.Float64Flag{
+				Name:        "score, s",
+				Usage:       "change the beacon cutoff score",
+				Destination: &cutoffScore,
+				Value:       .7,
+			},
 		},
 		Action: showBeacons,
 	}
 
 	bootstrapCommands(command)
 }
 
-type bcnresult struct {
```

Comment:
We really need to focus on using the data structures in our datastructure package!!!